homeassistant/components/media_player/webostv.py

```diff
@@ -240,7 +240,7 @@ def media_content_type(self):
     def media_image_url(self):
         """Image url of current playing media."""
         if self._current_source_id in self._app_list:
-            return self._app_list[self._current_source_id]['largeIcon']
+            return self._app_list[self._current_source_id]['icon']
```

Comment:
in webos v2 largeIcon is a url and looks better so I suggest maybe check if largeIcon is not a url fallback to icon?

e.g.
```python
if self._current_source_id in self._app_list:
  icon = self._app_list[self._current_source_id]['largeIcon']
  if not icon.startswith('http'):
    icon = self._app_list[self._current_source_id]['icon']
  return icon
return None
```

any thoughts?